grammars/.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "overrides": [{ "files": "*.json", "options": { "tabWidth": 4 } }]
```

Comment:
Ideally, unrelated changes shouldn't be included, but this looks reasonable, so I'd be fine with adding a Prettier config. Could you format it in a similar way to the other JSON files with line-breaks and indents (not sure if the config itself has to be 2-space-indented)?